upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -141,6 +141,25 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 		}
 	}
 
+	if featureflag.PublicJWKS.Enabled() {
+		key := "anonymous-access.addons.k8s.io"
```

Comment:
Perhaps call it "anonymous-issuer-discovery"?

Reply:
Done!